packages/scss/src/components/_tooltip.scss

```diff
@@ -1,14 +1,14 @@
 .tooltip {
-	background: $grey;
+	background: _get($luTheme, "palette.colors.greyStandard");
 	border-radius: 3px;
 	color: white;
 	display: inline-block;
-	font-size: $font-small;
+	font-size: _get($luTheme, "font.size.small");
 	padding: .5rem 1rem;
 	position: relative;
 
 	&::before {
-		border-color: $grey transparent transparent transparent;
+		border-color: _get($luTheme, "palette.colors.greyStandard") transparent transparent transparent;
```

Comment:
The triangle making mixin in Lucca-ui could have been kinda useful here. Same reason, many developers won't remember the css triangle trick.